### DISCUSSION TITLE
Creating a long-running career coach app (fine tuning?)

### DISCUSSION THREAD

**Discussion by markpeace** [langchain-ai/langsmith-sdk#684] [Q&A]
I’m interested to create an app which acts as a career coach as I transition to a new role. I want to have an ongoing conversation about my work and for the app to learn about it as I go along so that it can provide advice.

It works quite nicely as a conversation with memory, but of course the context window fills up eventually. I’ve tried to fine-tune the model using the conversation data provided so far, but it seems to forget all the details when I do this.

Any suggestion for where I’m going wrong?

**Comment by hinthornw** [comment]
Do you want to try out LangMem? It's what powers the demo journaling app: https://journal.langchain.com/journal

**Reply by markpeace** [reply]
Yes - this looks like the kind of thing I'm after ... where can I learn more?

**Reply by markpeace** [reply]
@hinthornw - just wondering how I get access to this? Cheers

**Reply by hinthornw** [reply]
Could you email me (will@langchain.dev) to get set up? Thanks!
